README.md

```diff
@@ -46,12 +46,11 @@ ESPResSo++ requires Python 3.7 or newer. All required Python packages are listed
 $ cd espressopp
 $ cmake .
 $ make -j2
```

Comment:
Should `make install` also be called?

Reply:
It was not mentioned there before...maybe something we should add in #395.

This part needs a clean-up, too, as in-source builds are really not a thing anymore.